.gitignore

```
@@ -46,3 +46,4 @@ phpstan.neon
 /phpunit/files/
 /phpunit/config
 *.phar
+.env
```

```suggestion
.env
.devcontainer/docker-compose.override.yaml
```

Is a personal and local file

---

indeed, It was not visible in the snippet of the diff